core/google/api/core/retry.py

```diff
@@ -77,7 +125,8 @@ def exponential_sleep_generator(
             be greater than 0.
         maximum (float): The maximum about of time to delay.
         multiplier (float): The multiplier applied to the delay.
-        jitter (float): The maximum about of randomness to apply to the delay.
+        max_jitter (float): The maximum about of randomness to apply to the
```

Comment:
I am sorry. I even thought about it at the time, but decided then that "he has a good reason". I should have asked then.